doc/ref_cert/lfn/chapters/chapter04.md

```diff
@@ -1,21 +1,75 @@
 [<< Back](../)
 
-# 4. NFVI Test Case Traceability to CNTT Requirements
+# 4. NFVI Test Case Traceability to Architecture Requirements
 <p align="right"><img src="../figures/bogo_ifo.png" alt="scope" title="Scope" width="35%"/></p>
 
 ## Table of Contents
 * [4.1 Introduction](#4.1)
-* [4.2 Heading](#4.2)
-* [4.3 Heading](#4.3)
+* [4.2 Selection Criteria](#4.2)
+* [4.3 Traceability Matrix](#4.3)
+  * [4.3.1 Architecture and OpenStack Based](#4.3.1)
+  * [4.3.2 Infrastructure](#4.3.2)
+  * [4.3.3 VIM](#4.3.3)
+  * [4.3.4 Interfaces & APIs](#4.3.4)
+  * [4.3.5 Tenants](#4.3.5)
+  * [4.3.6 LCM](#4.3.6)
+  * [4.3.7 Assurance](#4.3.7)
+  * [4.3.8 Security](#4.3.8)
 
 <a name="4.1"></a>
 ## 4.1 Introduction
 
-> Scope of this chapter is to have a list of test cases needed (a detailed table of sort)
+The scope of this chaptor is to identify and list down testcases based on <a href=https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/openstack/chapters/chapter02.md>requirements for reference archtiecture-1 (OpenStack based)</a>. This will serve as traceability between test cases and requirements.
```

Comment:
chaptor -> chapter
Please use md linking instead of raw html.

Reply:
Updated/corrected using MD vs. HTML. For future authors, here's a link to basic MD syntax: https://www.markdownguide.org/basic-syntax/. Can you review/Close @CsatariGergely ?